### PR TITLE
fix: show mime type icon for attachments in messages

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

Might not be the best solution, but as a workaround.

@Ivansss @mahibi Do you use all available icons for mime types from server, including theming?

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

### CODE REVIEW

**Comment by Ivansss** [comment]
Most of them but not all of them.
In Talk iOS we need to update them to the latest version of them https://github.com/nextcloud/talk-ios/issues/1578

File: src/shared/globals/globals.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ import { translate, translatePlural } from '@nextcloud/l10n'
 
 import { appData } from '../../app/AppData.js'
 import { dialogs } from './OC/dialogs.js'
+import { MimeTypeList } from './OC/MimeType.js'
```
Comment (Antreesy) [inline src/shared/globals/globals.js:27]: incorrect filename, buiuld is failing

**Review by Antreesy — APPROVED** [review]
After fixing the import, works correctly (tested against main and stable27)

**Review by DorraJaouad — APPROVED** [review]
Tested 👍🏻 
after filename change in import

**Comment by mahibi** [comment]
> @Ivansss @mahibi Do you use all available icons for mime types from server, including theming?

See https://github.com/nextcloud/talk-android/blob/f5a4274e545f506861dc7baf7f92ec5e73d674f6/app/src/main/java/com/nextcloud/talk/utils/DrawableUtils.kt for android, did not compare..
Some were updated by https://github.com/nextcloud/talk-android/pull/3719

File: src/shared/globals/globals.js (hunk 1 above)
Comment (ShGKme) [reply src/shared/globals/globals.js]: I though `eslint-pluing-import` should have caught this 🙈

**Comment by ShGKme** [comment]
- Fixed import
- Rebased
- Not using any custom theming from server, mobile clients also don't use them

**Pull request merged**